controllers/applicationset_controller.go

```diff
@@ -19,9 +19,10 @@ package controllers
 import (
 	"context"
 	"fmt"
+
 	"github.com/argoproj/argo-cd/v2/pkg/apiclient"
 	argoclient "github.com/argoproj/argo-cd/v2/pkg/apiclient"
-	appset "github.com/argoproj/argo-cd/v2/pkg/apis/applicationset/v1alpha1"
+	appset "github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1"
```

Comment:
A change in import path is a bit unsettling, we might want to check it before merging

Reply:
Seems to be working, the pod is up and no restarts

Reply:
This was changed because the original API "github.com/argoproj/argo-cd/v2/pkg/apis/applicationset/v1alpha1" is not present for ArgoCD 2.5 (Refer https://pkg.go.dev/github.com/argoproj/argo-cd/v2@v2.5.7)
<img width="451" alt="image" src="https://user-images.githubusercontent.com/214651/213633771-388c441c-9bd5-4bde-8e56-26df1369eba2.png">